### ISSUE TITLE
[Linux] desktop file is lacking a Categories= entry

### ISSUE THREAD

**Issue by probonopd** [DeDop/dedop-studio#3]
The desktop file is lacking a `Categories=` entry in the `*.desktop` file.

According to the [menu spec](https://standards.freedesktop.org/menu-spec/latest/apa.html),

> By including one of the Main Categories in an application's desktop entry file, the application will be ensured that it will show up in a section of the application menu dedicated to this category. If multiple Main Categories are included in a single desktop entry file, the entry may appear more than once in the menu. 

Hence, please add at least one of the following in the `Categories=` key.

Main Category | Description | Notes
-- | -- | --
AudioVideo | Application for presenting, creating, or processing multimedia (audio/video) |  
Audio | An audio application | Desktop entry must include AudioVideo as well
Video | A video application | Desktop entry must include AudioVideo as well
Development | An application for development |  
Education | Educational software |  
Game | A game |  
Graphics | Application for viewing, creating, or processing graphics |  
Network | Network application such as a web browser |  
Office | An office type application |  
Science | Scientific software |  
Settings | Settings applications | Entries may appear in a separate menu or as part of a 		"Control Center"
System | System application, "System Tools" such as say a log viewer or network monitor |  
Utility | Small utility application, "Accessories"

In addition, you could specify one or more from the longer list of [Additional Categories](https://standards.freedesktop.org/menu-spec/latest/apas02.html).

Please test the result with `desktop-file-validate` and make sure it passes.

electron-builder offers native support for this since [v19.22.1](https://github.com/electron-userland/electron-builder/releases/tag/v19.22.1).

Reference: https://github.com/AppImage/AppImageHub/issues/2

**Comment by probonopd** [comment]
If someone could be so nice and ping me once a release is available on GitHub Releases that includes this fix, I could add it to the AppImageHub central directory of available AppImages. Thanks!

**Comment by hans-permana** [comment]
hi @probonopd , this fix has been included in DeDop Studio 1.2.0, which is available [here](https://github.com/DeDop/dedop-studio/releases/tag/v1.2.0). Thanks.

**Comment by probonopd** [comment]
Thanks, added
https://github.com/AppImage/appimage.github.io/commit/470343493589009e2f90a585be10b638659d564a

However, I could not get a screenshot, possibly related to this error:

```
A JavaScript error occurred in the main process
Uncaught Exception:
Error: ENOENT: no such file or directory, open '/home/travis/.dedop/dedop.log'
    at Error (native)
    at Object.fs.openSync (fs.js:640:18)
    at Object.module.(anonymous function) [as openSync] (ELECTRON_ASAR.js:173:20)
    at Object.fs.writeFileSync (fs.js:1343:33)
    at Object.fs.appendFileSync (fs.js:1402:6)
    at writeToLogFile (/run/firejail/appimage/.appimage-7324/usr/bin/resources/app.asar/main/main.js:182:8)
    at Object.init (/run/firejail/appimage/.appimage-7324/usr/bin/resources/app.asar/main/main.js:185:5)
    at Object.<anonymous> (/run/firejail/appimage/.appimage-7324/usr/bin/resources/app.asar/main.js:2:6)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
```

Reference:
https://travis-ci.org/AppImage/appimage.github.io/builds/293792285#L708-L720
